<ui version="4.0" >
 <class>compositionView</class>
 <widget class="QDialog" name="compositionView" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Composition View</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="4" column="1" >
    <widget class="QToolButton" name="collapseButton" >
     <property name="text" >
      <string>+/-</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QToolButton" name="fitButton" >
     <property name="text" >
      <string>Fit</string>
     </property>
    </widget>
   </item>
   <item rowspan="5" row="0" column="0" >
    <widget class="QGraphicsView" name="graphicsView" >
     <property name="windowModality" >
      <enum>Qt::NonModal</enum>
     </property>
     <property name="resizeAnchor" >
      <enum>QGraphicsView::NoAnchor</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QToolButton" name="zoomInButton" >
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset>../resources/zoomin.png</iconset>
     </property>
     <property name="autoRepeat" >
      <bool>true</bool>
     </property>
     <property name="autoRepeatDelay" >
      <number>0</number>
     </property>
     <property name="autoRepeatInterval" >
      <number>33</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QSlider" name="verticalSlider" >
     <property name="maximum" >
      <number>500</number>
     </property>
     <property name="value" >
      <number>250</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition" >
      <enum>QSlider::TicksBelow</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QToolButton" name="zoomOutButton" >
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset>../resources/zoomout.png</iconset>
     </property>
     <property name="autoRepeat" >
      <bool>true</bool>
     </property>
     <property name="autoRepeatDelay" >
      <number>0</number>
     </property>
     <property name="autoRepeatInterval" >
      <number>33</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../solvergui.qrc" />
 </resources>
 <connections/>
</ui>
